packages/main/src/plugin/api/onboarding.ts

```diff
@@ -29,6 +29,7 @@ export interface OnboardingStepItem {
 
 export type OnboardingStatus = 'completed' | 'failed' | 'skipped';
 export type OnboardingState = 'completed' | 'failed';
+export type OnboardingEmbeddedComponentId = 'create';
```

Comment:
it looks like it's more a type of available components

like `OnboardingEmbeddedComponentType` where you should have `createContainerProviderConnection` or `createKubernetesProviderConnection`